src/debug/ee/functioninfo.cpp

```diff
@@ -1580,6 +1580,13 @@ DebuggerJitInfo *DebuggerMethodInfo::FindOrCreateInitAndAddJitInfo(MethodDesc* f
         _ASSERTE(g_pEEInterface->GetNativeCodeMethodDesc(startAddr) == fd);
     }
 
+#ifdef _TARGET_ARM_
+    if (startAddr != NULL)
+    {
+        startAddr |= THUMB_CODE;
```

Comment:
`PCODE` is expected to have the thumb bit set: https://github.com/dotnet/coreclr/blob/master/src/inc/daccess.h#L2416

It would be more appropriate to fix the places that casts the value to PCODE to set the thumb bit.

Reply:
(Nit: Could you please fix the `should should` typo in daccess.h while you are on it?)